ccd-definition/AuthorisationCaseField/court-admin.json

```diff
@@ -356,6 +356,13 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
   },
+  {
```

Comment:
small, but this diff isn't necessary if we move it back to where it was originally

Reply:
dunno how that happened 😄 reverting